README.rst

```diff
@@ -1,254 +1,22 @@
-* ``use_spaces`` enforces spaces over tabs in the codebase. ``use_spaces`` is a
-  setting for the ``SpaceConsistencyBear`` defined
-  `here <https://github.com/coala/bear-docs/blob/master/docs/SpaceConsistencyBear.rst>`__.
-
-``[Spacing]`` is a *section*. Sections are executed in the order you
-define them.
-
-Store the file in the project's root directory and run coala:
-
-.. code-block:: bash
-
-    $ coala
-
-Please read our
-`coafile specification <http://docs.coala.io/en/latest/Users/coafile.html>`__
-to learn more.
-
-Using command-line arguments
-****************************
-
-However, if you don't want to save your settings, you can also run coala with
-command line arguments:
-
-.. code-block:: bash
-
-    $ coala --files=setup.py --bears=SpaceConsistencyBear -S use_spaces=True
-
-Note that this command does the same thing as having a coafile and running
-`coala`. The advantage of having a coafile is that you don't need to enter the
-settings as arguments everytime.
-
-To get the complete list of arguments and their meaning, run:
-
-.. code-block:: bash
-
-    $ coala --help
-
-You can find a quick demo of coala here:
-
-|asciicast|
-
-.. |asciicast| image:: https://cdn.rawgit.com/coala/artwork/master/artwork/asciinema_demo/asciinema_demo.png
-   :target: https://asciinema.org/a/42968?autoplay=1
-   :width: 100%
-
------
-
-======
-Awards
-======
+|codecov.io| |Documentation Status| |Gitmate| |AGPL| |Chat|
 
 .. image:: http://www.yegor256.com/images/award/2016/winner-sils.png
    :alt: Awards - Yegor256 2016 Winner
 
------
-
-================
-Getting Involved
-================
-
-If you would like to be a part of the coala community, you can check out our
-`Getting In Touch <http://docs.coala.io/en/latest/Help/Getting_In_Touch.html>`__
-page or ask us at our active Gitter channel, where we have maintainers from
-all over the world. We appreciate any help!
-
-We also have a
-`newcomer guide <http://coala.io/newcomer>`__
-to help you get started by fixing an issue yourself! If you get stuck anywhere
-or need some help, feel free to contact us on Gitter or drop a mail at our
-`newcomer mailing list <https://groups.google.com/d/forum/coala-newcomers>`__.
-
-|gitter|
-
------
-
-=======
-Support
-=======
-
-Feel free to contact us at our `Gitter channel <https://gitter.im/coala/coala>`__, we'd be happy to help!
-
-You can also drop an email at our
-`mailing list <https://github.com/coala/coala/wiki/Mailing-Lists>`__.
-
------
-
-=======
-Authors
-=======
-
-coala is maintained by a growing community. Please take a look at the
-meta information in `setup.py <setup.py>`__ for the current maintainers.
-
------
-
-=======
-License
-=======
-
-|AGPL|
-
-.. |Windows| image:: https://img.shields.io/badge/platform-Windows-brightgreen.svg
-.. |Linux| image:: https://img.shields.io/badge/platform-Linux-brightgreen.svg
-.. |Stable| image:: https://img.shields.io/badge/latest%20stable-0.9.0-green.svg
```

Comment:
you mean you want to keep the version badge? I'm against that because we continue to forget to update it and the version is there on all platforms: github has the releases, pypi as well.